src/ducks/Studio/url.js

```diff
@@ -8,7 +8,20 @@ const { trendPositionHistory } = Events
 const COMPARE_CONNECTOR = '-CC-'
 const getMetricsKeys = metrics => metrics.map(({ key }) => key)
 const toArray = keys => (typeof keys === 'string' ? [keys] : keys)
-const convertKeyToMetric = (key, dict) => dict[key] || compatabilityMap[key]
+const convertKeyToMetric = (key, dict) =>
+  dict[key] || compatabilityMap[key] || getFromSubmetrics(key, dict)
+
+const getFromSubmetrics = (key, metrics) => {
+  for (let metricKey in metrics) {
```

Comment:
It's a bad practice to use `for .. in`  for arrays. Use `for` loop or `forEach`, `for .. of` iterator

Reply:
@DmitriVanGuard , metrics is 'object', not array.

Reply:
Objects should not follow array naming conventions, otherwise it leads to implicit errors